bots/jabber_mirror_backend.py

```diff
@@ -298,32 +298,31 @@ def config_error(msg):
 
 if __name__ == '__main__':
     parser = optparse.OptionParser(epilog=
-'''Most general and Jabber configuration options may also be specified in the
-zulip configuration file under the jabber_mirror section (exceptions are noted
-in their help sections).  Keys have the same name as options with hyphens
-replaced with underscores.  Zulip configuration options go in the api section,
-as normal.'''.replace("\n", " ")
-                                    )
+                                   '''Most general and Jabber configuration options may also be specified in the
+                                   zulip configuration file under the jabber_mirror section (exceptions are noted
+                                   in their help sections).  Keys have the same name as options with hyphens
+                                   replaced with underscores.  Zulip configuration options go in the api section,
+                                   as normal.'''.replace("\n", " "))
```

Comment:
(Just a clarification: In order to concatenate the strings that span multiple lines you will have to enclose them in brackets.)